examples/scripts/sentiment_tuning.py

```diff
@@ -197,7 +197,7 @@ def collator(data):
     # Compute sentiment score
     texts = [q + r for q, r in zip(batch["query"], batch["response"])]
     pipe_outputs = sentiment_pipe(texts, **sent_kwargs)
-    rewards = [torch.tensor(output[1]["score"]) for output in pipe_outputs]
+    rewards = [torch.tensor(output[1]["score"], device=device) for output in pipe_outputs]
```

Comment:
I think you don't need this change. The `_step_safety_checker` puts all the tensors on the right device